compiler/rustc_session/src/config/cfg.rs

```diff
@@ -143,6 +143,7 @@ pub(crate) fn disallow_cfgs(sess: &Session, user_cfgs: &Cfg) {
             | (sym::target_has_atomic_load_store, Some(_))
             | (sym::target_thread_local, None) => disallow(cfg, "--target"),
             (sym::fmt_debug, None | Some(_)) => disallow(cfg, "-Z fmt-debug"),
+            (sym::emscripten_wasm_eh, None | Some(_)) => disallow(cfg, "-Z emscripten_wasm_eh"),
```

Comment:
Please add a test for the cfg in https://github.com/rust-lang/rust/blob/f55a5d2e067629c7939a7d8373e19dc673552934/tests/ui/cfg/disallowed-cli-cfgs.rs